Makefile

```diff
@@ -164,7 +163,7 @@ publish:
 	)
 
 check_codegen:
-	command $(CODEGEN) > /dev/null 2>&1 || { echo $CODEGEN_INSTALL_MSG; false; }
+	test -s ${SWAGGER_JAR} || wget  ${SWAGGER_DOWNLOAD_URL} -O ${SWAGGER_JAR}
```

Comment:
This seemed like the simplest approach, but I'm not a Makefile expert so I'm open to suggestions.

Reply:
I think it would make sense to copy to make the local jar location a target of the Makefile (and listed in the .gitignore).  Then the rule would just download it.

Reply:
I'm not sure I follow your latest comment. Isn't `check_codegen` already a target? Also there is a new entry in `.gitignore` for the jar